README.md

```diff
@@ -65,7 +69,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Label PRs
-        uses: tinkurlab/monorepo-pr-labeler-action
+        uses: tinkurlab/monorepo-pr-labeler-action@master
```

Comment:
we should mention latest version here

Reply:
"we should mention latest version here" as in the specific version ex `3.1.1`?